src/components/WorldMap.js

```diff
@@ -136,13 +142,15 @@ export class WorldMap extends Component {
       countries: mapData.features,
     });
 
+    let locationSet='false';
```

Comment:
Can you use booleans here instead of strings?

Reply:
@LarsDenBakker  corrected. Can you approve now